Libraries/Components/TextInput/TextInput.js

```diff
@@ -333,6 +334,31 @@ type IOSProps = $ReadOnly<{|
   textContentType?: ?TextContentType,
 |}>;
 
+// [TODO(macOS GH#774)
+export type SubmitKeyEvent = $ReadOnly<{|
+  key: string,
+  altKey?: ?boolean,
+  ctrlKey?: ?boolean,
+  metaKey?: ?boolean,
+  shiftKey?: ?boolean,
+  functionKey?: ?boolean,
+|}>;
+
+type MacOSProps = $ReadOnly<{|
```

Comment:
I should have done this before.
I'll make another PR to consolidate once my other changes landed

Reply:
I've been wanting to clean up some of the flow types of View as well so that we have a ViewPropsMacOS too or something like that. Appreciate the change :)